lib/util/DataURI.js

```diff
@@ -0,0 +1,32 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+*/
+
+"use strict";
+
+// data URL scheme: "data:text/javascript;charset=utf-8;base64,some-string"
+// http://www.ietf.org/rfc/rfc2397.txt
+const URIRegEx = /^data:([^;,]+)?((?:;(?:[^;,]+))*?)(;base64)?,(.*)$/;
```

Comment:
/cc @goganchic I think we forget `i` flag here, `data` protocol is case insensitivity

Reply:
added in https://github.com/webpack/webpack/pull/11095